sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Release History
 
 ## 4.0.0b4 (Unreleased)
+### Added
+- Added `KeyVaultRoleAssignmentProperties` class
```

Comment:
Wasn't this in the last version?

Reply:
It was, that's true -- it just wasn't directly used in any way since `KeyVaultRoleAssignment` flattened the properties. The other changelog bullet about this change covers that, so I can remove this. Thanks!